omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ObsController1_9Test.java

```diff
@@ -84,6 +87,38 @@ public void shouldGetAll() throws Exception {
 	}
 	
 	@Test
+	public void shouldGetallIncludingVoidedObs() throws Exception {
+		
+		executeDataSet("encounterWithObsGroup1_9.xml");
+		
+		MockHttpServletRequest allNonVoidedObsRequest = newGetRequest(getURI());
+		allNonVoidedObsRequest.addParameter("encounter", "62967e68-96bb-11e0-8d6b-9b9415a91465");
+		MockHttpServletResponse allNonVoidedObsResponse = handle(allNonVoidedObsRequest);
+		List<Object> allNonVoidedObsList = deserialize(allNonVoidedObsResponse).get("results");
+		
+		assertEquals(allNonVoidedObsList.size(), 6);
+		
+		MockHttpServletRequest deleteRequest = newDeleteRequest(getURI() + "/" + "11de743c-96cd-11e0-8d6b-9b9415a91465");
+		deleteRequest.addParameter("reason", "test voided obs");
+		handle(deleteRequest);
+		
+		MockHttpServletRequest allObsIncludingVoidedRequest = newGetRequest(getURI());
+		allObsIncludingVoidedRequest.addParameter("encounter", "62967e68-96bb-11e0-8d6b-9b9415a91465");
```

Comment:
Can you avoid using this uuid from more than one place 62967e68-96bb-11e0-8d6b-9b9415a91465?

Reply:
@dkayiwa 
I am not sure I follow what you mean since the uuids above are different;
62967e68-96bb-11e0-8d6b-9b9415a91465 is the  encounter uuid
11de743c-96cd-11e0-8d6b-9b9415a91465 is the observation uuid

Reply:
Let me put it this way, are you aware of the fact that you are using 62967e68-96bb-11e0-8d6b-9b9415a91465 in more than one place?

Reply:
This has been corrected by assigning the uuid to a variable.